src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -19,7 +19,22 @@ public static PatchMap Generate(WorldGenerationSettings settings, Species defaul
 
         // Initialize the graph's random parameters
         var regionCoordinates = new List<Vector2>();
-        int vertexCount = random.Next(6, 10);
+
+        int vertexCount = 4;
+
+        if (settings.WorldSize == WorldGenerationSettings.WorldSizeEnum.Small)
```

Comment:
I think this should use a switch statement and print an error if the value is unhandled.